engine/src/test/java/org/pentaho/di/job/entries/copyfiles/JobEntryCopyFilesTest.java

```diff
@@ -151,22 +153,20 @@ public void saveLoad() throws Exception {
 
   @Test
   public void saveLoadWithPassword() throws Exception {
-    String[] srcPath = new String[] { "EMPTY_SOURCE_URL-0-hdfs://user321:321fake@foo.bar.com:8020/user/user321" };
-    String[] destPath = new String[] { "EMPTY_DEST_URL-0-hdfs://user123:fake123@foo.bar.com:8020/user/user123" };
-
-    String srcPathScrubbedEncoded = "EMPTY_SOURCE_URL-0-hdfs://user321:&lt;password&gt;@foo.bar.com:8020/user/user321";
-    String destPathScrubbedEncoded = "EMPTY_DEST_URL-0-hdfs://user123:&lt;password&gt;@foo.bar.com:8020/user/user123";
+    String srcPath = "EMPTY_SOURCE_URL-0-hdfs://user321:321fake@foo.bar.com:8020/user/user321";
```

Comment:
purposely allowing hardcoded URLs, this test verifies this, see "Technical Direction" section of jira comment : https://hv-eng.atlassian.net/browse/PDI-19665?focusedCommentId=2033576